test/units/docker_registry_test.rb

```diff
@@ -18,7 +18,9 @@ class DockerRegistryTest < ActiveSupport::TestCase
   end
 
   test 'password is stored encrypted' do
-    registry = as_admin { FactoryGirl.create(:docker_registry) }
+    registry = as_admin { FactoryGirl.build(:docker_registry) }
+    registry.password = 'encrypted-whatever'
```

Comment:
Setting it to encrypted- means you might not actually be testing that it encrypts a password, only that it offers to decrypt one that's already encrypted...  I think this line should be removed, then either:

1. move the stubbing to above the registry object creation, so when `password=` is called by the factory, it's encrypted, or
2. set the password after stubbing